src/components/ClusteredMultiselect.vue

```diff
@@ -164,13 +168,16 @@ export default {
 			selectAllValue: false,
 			queryString: '',
 			indeterminate: false,
+			uniqueKey: generateKey(),
 		};
 	},
 
 	computed: {
 		selectedFancyMessage() {
 			return (qty) => {
-				if (qty === 1) return '1 opção selecionada';
+				if (qty === 1) {
```

Comment:
Não acho que precisa mudar, se for o caso, agora, mas só a nível de dúvida: aqui a ideia era ser "qty" mesmo ou "qtd"